bears/python/PyImportSortBear.py

```diff
@@ -182,7 +184,31 @@ def run(self, filename, file,
 
         if new_file != tuple(file):
             diff = Diff.from_string_arrays(file, new_file)
+            all_imports = []
+            for line_nr in range(diff.range(filename).start.line,
+                                 diff.range(filename).end.line + 1):
+                if(diff._get_change(line_nr).change is not False):
+                    line_diff = diff._get_change(line_nr).change[0]
+                    list_of_imports = re.split(', +', line_diff.rstrip())
+                    if 'import' in list_of_imports[0]:
+                        list_of_imports[0] = list_of_imports[0][
+                            list_of_imports[0].find('import') + 7:]
+                        list_of_imports.sort()
+                        string_of_imports = ' '.join(list_of_imports)
+                        all_imports.append(string_of_imports)
+                elif(diff._get_change(line_nr).delete is not False):
+                    line_diff = diff.original[line_nr - 1]
+                    list_of_imports = re.split(', +', line_diff.rstrip())
+                    if 'import' in list_of_imports[0]:
+                        list_of_imports[0] = list_of_imports[0][
+                            list_of_imports[0].find('import') + 7:]
+                        string_of_imports = ' '.join(list_of_imports)
+                        all_imports.append(string_of_imports)
+                        all_imports.sort()
+                    else:
+                        all_imports.append('')
```

Comment:
line_nr.change will only be true if it is changed. Therefore that condition is checked.